src/backend/setup.cfg

```diff
@@ -0,0 +1,22 @@
+;;
+;; Third-party packages configuration
+;;
+[flake8]
+max-line-length = 99
+exclude =
+    .git,
+    .venv,
+    build,
+    venv,
+    __pycache__,
+    node_modules,
+    */migrations/*
+
+[isort]
+known_django=django
+known_richie=richie
+include_trailing_comma=True
+line_length=88
+multi_line_output=3
+sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,RICHIE,FIRSTPARTY,LOCALFOLDER
```

Comment:
Here `richie` is a third party and `funmooc` should be isolated?

Reply:
I had hesitations on this point. You are probably right. I'll fix this.

Reply:
Now we consider that Richie must be a third party app, and we will use relative import for `funmooc` (_i.e._ project-related) imports.